src/_tests/test_train.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pytest
+from tiled.catalog import from_uri
```

Comment:
I ran into one error when initiating the `pytest` in command line. And I've also tried to import `from_uri` from `tiled.client` instead, it did not go through either.

Traceback:

```
src/_tests/test_train.py:3: in <module>
    from tiled.catalog import from_uri
../../miniforge3/envs/dlsia_prototype_pr23/lib/python3.9/site-packages/tiled/catalog/__init__.py:1: in <module>
    from .adapter import from_uri, in_memory  # noqa: F401
../../miniforge3/envs/dlsia_prototype_pr23/lib/python3.9/site-packages/tiled/catalog/adapter.py:37: in <module>
    from ..server.schemas import DataSource, Management, Revision, Spec
../../miniforge3/envs/dlsia_prototype_pr23/lib/python3.9/site-packages/tiled/server/schemas.py:85: in <module>
    Specs = pydantic.conlist(Spec, max_items=20)
E   TypeError: conlist() got an unexpected keyword argument 'max_items'
```

Reply:
This PR adds to `tiled[all]` to `requirements-dev.txt". My guess is that you have not re-run a pip install of that file.

Reply:
No I did pip install that file, but I wonder if it got overwritten by the tiled[client]. When I was testing, I first created a fresh conda env and pip installed `requirements-dev.txt`, then I though I might need to install dlsia and torch for local testing, so I pip installed `requirements.txt` which only contains `tiled[client]`, this might be the cause.

I will try to reinstall the `requirements-dev.txt` to see if it works.

Reply:
> No I did pip install that file, but I wonder if it got overwritten by the tiled[client]. When I was testing, I first created a fresh conda env and pip installed `requirements-dev.txt`, then I though I might need to install dlsia and torch for local testing, so I pip installed `requirements.txt` which only contains `tiled[client]`, this might be the cause.
> 
> I will try to reinstall the `requirements-dev.txt` to see if it works.

Update: reinstall `requirements-dev.txt` solved the issue, my guess here is probably true: the `tiled[client]` package will overwrite `tiled[all]` and took out server side functionalities.